person.rb

```diff
@@ -3,14 +3,15 @@
 class Person
   attr_accessor :name, :age
 
-  attr_reader :id
+  attr_reader :id, :rentals
```

Comment:
You need to add `:rentals` to your `attr_accessor` collection so that `Rental` class (/rental.rb line 9) be able to access that variable, read and set it to a new value. 👍🏽

Reply:
Yes